### ISSUE TITLE
bookmarks: Rename "Add bookmark"?

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit-files#657]
>  I think the "Add bookmark" label is a bit confusing. I almost thought it would open a new dialog. Something like "Add to bookmarks" (similar to Nautilus) may be better.

_Originally posted by @subhoghoshX in https://github.com/cockpit-project/cockpit-files/issues/522#issuecomment-2223347959_

**Comment by garrett** [comment]
Alternatives:

"Bookmark current directory"
"Add current directory"

However, the opposite would be "Remove current directory", which sounds like `rm -rf`, unless it's a really lengthy "Remove current directory from bookmarks".

We're currently using "Remove current directory", so it's either fine enough through context, or we should change that too.

**Comment by Mash707** [comment]
@garrett I feel "Remove from bookmarks" seems to be a better option. For a new user "Remove current directory" could relate to deleting the directory.

**Comment by garrett** [comment]
@Mash707: Yep, definitely. That's exactly the point I was making. I'm happy it makes sense to you too.

I think this should be a really easy change (I think it's just making sure there the bookmark actions are "Add to bookmarks" and "Remove from bookmarks") and it looks like it "fell through the cracks" (that is: we were all busy with a lot of other tasks and overlooked this one on accident).

Would you like to send a PR?

**Comment by Mash707** [comment]
Sure I'll send a PR ASAP.

**Comment by garrett** [comment]
You can take your time, but it'll be a welcome PR. Hopefully it'll be super easy and will be reviewed quickly. Thanks!
